content/deployment/quick_start.md

```diff
@@ -79,7 +79,7 @@ Note that while it's common practice to make a new `modsecurity.d` folder, as ou
 
 ```bash
 mkdir /etc/httpd/modsecurity.d
-tar -zxvf v{{< param crs_latest_release >}}.tar.gz -C /etc/httpd/modsecurity.d/owasp-modsecurity-crs
+tar -zxvf v{{< param crs_latest_release >}}.tar.gz -C /etc/httpd/modsecurity.d/coreruleset
```

Comment:
```suggestion
tar -zxvf v{{< param crs_latest_release >}}.tar.gz -C /etc/httpd/modsecurity.d/crs
```
It's easier if you add a suggestion in the code itself. Less ambiguous, easy to follow.